Makefile

```diff
@@ -45,11 +45,17 @@ all build:
 #
 # Example:
 #   make build-tests
-build-tests:
-	hack/build-go.sh test/extended/extended.test
-	hack/build-go.sh test/integration/integration.test -tags='integration docker'
+build-tests: build-extended-test build-integration-test
```

Comment:
I'd like it to be as frictionless as possible for newcomers to do write test / compile / run / fix; splitting this into two pieces helps.  Also this is referenced by the attached documentation update.